pkg/agent/flowexporter/broadcaster/interface.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package broadcaster
+
+import "antrea.io/antrea/pkg/agent/flowexporter/connection"
+
+type Subscriber interface {
+	Subscribe() *subscription
+	Unsubscribe(*subscription)
+}
+
+type Publisher interface {
+	Publish(conns []*connection.Connection)
+}
+
+type Broadcaster interface {
+	Subscriber
+	Publisher
+	Start(stopCh <-chan struct{})
```

Comment:
We usually use `Run` for this method name
We also usually exclude the method from the interface name, is that required here?

Reply:
Removed, using the channels util now.